SPECS/shim-unsigned-aarch64/shim-unsigned-aarch64.spec

```diff
@@ -1,19 +1,83 @@
+Patch0019: 0019-Cryptlib-fix-build-on-32bit-ARM.patch
+Patch0020: 0020-Make-sure-that-MOK-variables-always-get-mirrored.patch
+Patch0021: 0021-mok-fix-the-mirroring-of-RT-variables.patch
+Patch0022: 0022-mok-consolidate-mirroring-code-in-a-helper-instead-o.patch
+Patch0023: 0023-shim-only-include-shim_cert.h-in-shim.c.patch
+Patch0024: 0024-mok-also-mirror-the-build-cert-to-MokListRT.patch
+Patch0025: 0025-mok-minor-cleanups.patch
+Patch0026: 0026-Remove-call-to-TPM2-get_event_log.patch
+Patch0027: 0027-Make-EFI-variable-copying-fatal-only-on-secureboot-e.patch
+Patch0028: 0028-VLogError-Avoid-NULL-pointer-dereferences-in-V-Sprin.patch
+Patch0029: 0029-Once-again-try-even-harder-to-get-binaries-without-t.patch
+Patch0030: 0030-shim-Rework-pause-functions-and-add-read_counter.patch
+Patch0031: 0031-Hook-exit-when-shim_lock-protocol-installed.patch
+Patch0032: 0032-Work-around-stuff-Waddress-of-packed-member-finds.patch
+Patch0033: 0033-Fix-a-use-of-strlen-instead-of-Strlen.patch
+Patch0034: 0034-MokManager-Use-CompareMem-on-MokListNode.Type-instea.patch
+Patch0035: 0035-OpenSSL-always-provide-OBJ_create-with-name-strings.patch
+Patch0036: 0036-Use-portable-shebangs-bin-bash-usr-bin-env-bash.patch
+Patch0037: 0037-tpm-Fix-off-by-one-error-when-calculating-event-size.patch
+Patch0038: 0038-tpm-Define-EFI_VARIABLE_DATA_TREE-as-packed.patch
+Patch0039: 0039-MokManager-console-mode-modification-for-hi-dpi-scre.patch
+Patch0040: 0040-MokManager-avoid-Werror-address-of-packed-member.patch
+Patch0041: 0041-tpm-Don-t-log-duplicate-identical-events.patch
+Patch0042: 0042-Slightly-better-debugging-messages.patch
+Patch0043: 0043-Actually-check-for-errors-from-set_second_stage.patch
+Patch0044: 0044-translate_slashes-don-t-write-to-string-literals.patch
+Patch0045: 0045-shim-Update-EFI_LOADED_IMAGE-with-the-second-stage-l.patch
+Patch0046: 0046-tpm-Include-information-about-PE-COFF-images-in-the-.patch
+Patch0047: 0047-Fix-the-license-on-our-buildid-extractor.patch
+Patch0048: 0048-Update-README.tpm.patch
+Patch0049: 0049-Check-PxeReplyReceived-as-fallback-in-netboot.patch
+Patch0050: 0050-Remove-a-couple-of-incorrect-license-claims.patch
+Patch0051: 0051-MokManager-fix-uninitialized-value.patch
+Patch0052: 0052-Fix-some-volatile-usage-gcc-whines-about.patch
+Patch0053: 0053-MokManager-fix-a-wrong-allocation-failure-check.patch
+Patch0054: 0054-simple_file-fix-uninitialized-variable-unchecked-ret.patch
+Patch0055: 0055-Fix-a-broken-tpm-type.patch
+Patch0056: 0056-Make-cert.S-not-impossible-to-read.patch
+Patch0057: 0057-Add-support-for-vendor_db-built-in-shim-authorized-l.patch
+Patch0058: 0058-Handle-binaries-with-multiple-signatures.patch
+Patch0059: 0059-Make-openssl-accept-the-right-set-of-KU-EKUs.patch
+Patch0060: 0060-Improve-debug-output-some.patch
+Patch0061: 0061-Also-use-a-config-table-to-mirror-mok-variables.patch
+Patch0062: 0062-Implement-lennysz-s-suggestions-for-MokListRT.patch
+Patch0063: 0063-hexdump.h-fix-arithmetic-error.patch
```

Comment:
That's quite a list of patches. Are they here to cover all security patches we're missing out on by having to use version 15 instead of 15.4?